packages/react-reconciler/src/__tests__/ReactIncrementalUpdates-test.js

```diff
@@ -474,11 +481,17 @@ describe('ReactIncrementalUpdates', () => {
 
     await ReactNoop.act(async () => {
       pushToLog('A');
-      Scheduler.unstable_runWithPriority(
+
+      // TODO: Double wrapping is temporary while we remove Scheduler runWithPriority.
+      ReactDOM.unstable_runWithPriority(
```

Comment:
@acdlite I don't think I can use ReactDOM here because the tests fail, but I also don't see how I can add it to the Noop. Any tips to fixing this last test?

Reply:
There are two pieces of context I'm missing:

- What is the rebasing behavior that this test is testing for?
- Is it ok for the Noop renderer to import ReactFiberLanes to implement a runWithPriority like ReactDOM has? I.e. what are the boundaries we want to enforce for the Noops renderer?

Reply:
- "Rebasing" occurs when a high pri update interrupts a lower priority update. React will commit the high pri update first. Then, when it renders the lower priority update, it needs to maintain the original order. So it replays the high pri update on top of the low pri update. That way the final order in which the updates are processed is the same as order that they were dispatched.